packages/turf-boolean-is-valid/index.js

```diff
@@ -0,0 +1,111 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
We would be adding a `.gitignore` which includes `index.js` to prevent this file from being uploaded